TourMate/TourMate/Frontend/Plan/Views/PlansViews/PlansViewModel.swift

```diff
@@ -148,3 +153,35 @@ extension PlansViewModel {
         self.isLoading = false
     }
 }
+
+// MARK: - View Methods
+extension PlansViewModel {
+    func makeSuggestionsView() -> some View {
+        let binding = Binding(
+            get: { self.isShowingTransportationOptionsSheet },
+            set: { self.isShowingTransportationOptionsSheet = $0 })
+
+        return VStack(alignment: .leading) {
+            Text("TOURMATE ASSISTANT").font(.subheadline)
+            ScrollView([.horizontal]) {
+                HStack {
+                    SuggestionButton(
+                        title: "Navigator",
+                        subtitle: "Search for transportation options",
+                        iconName: "car.fill") {
+                        self.isShowingTransportationOptionsSheet.toggle()
+                    }
+                        .sheet(isPresented: binding) {
+                            let viewModel = self.viewModelFactory.getTransportationOptionsViewModel(plans: self.plans)
+                            TransportationOptionsView(viewModel: viewModel)
+                        }
+
+                    SuggestionButton(title: "Conflict", subtitle: "There are clashing confirmed plans") {
+                        print("Implement conflict page")
+                    }
+                }
+            }
+        }
+        .padding()
```

Comment:
Why is it in the view model though?

Reply:
If we are going to do conflict checking or nearby places recommendation then there'll be quite a bit of logic that's why i push it to the viewModel.

Reply:
I don't think we should have view elements in view models actually. We can either shift it to some view util

Reply:
Alright i'll go and shift it